templates/begin_auth.tmpl.html

```diff
@@ -17,8 +17,16 @@ <h1 class="display-3">
                     Authentication Flow has begun!
                 </h1>
                 <h3>
-                    Add this to the text of one of your security questions:
-                    {{.Verifier}}
+                    Add this to the text of one of your security questions: {{.Verifier}}
+                    <br>
+                    To do that :
```

Comment:
We shouldn't put this information inside `div.jumbotron`. Put it after this div.